packages/markdown/src/index.test.ts

```diff
@@ -133,4 +133,28 @@ describe('Markdown processing', () => {
     const expected = '<p>testing</p>';
     await testMarkdown(question, expected);
   });
+
+  it('sanitizes inline HTML tags if allowHtml is false', async () => {
+    const question = 'testing <em>html</em>';
+    const expected = '<p>testing html</p>';
+    await testMarkdown(question, expected, { allowHtml: false });
+  });
+
+  it('sanitizes an HTML block if allowHtml is false', async () => {
+    const question = '<div>testing</div>';
+    const expected = '';
+    await testMarkdown(question, expected, { allowHtml: false });
```

Comment:
You probably want a test that checks that only the block is removed, not anything else, so it's probably a good idea to have some valid Markdown before and after the block.
```suggestion
    const question = '_Before_ the block\n\n<div>HTML block to be sanitized</div>\n\n**After** the block';
    const expected = '<p><em>Before</em> the block</p>\n<p><strong>After</strong> the block</p>';
    await testMarkdown(question, expected, { allowHtml: false });
```

I'm not 100% sure whether the renderer will put the line break before `</p>`, so you may need to adjust it.